docs/dashboards/installation.md

```diff
@@ -0,0 +1,34 @@
+Installation
+===
+
+### Install via NPM
+One of the ways of adding the Highcharts Dashboards to your web page is via npm script. You can get all Dashboards functionality in the package, which can be installed using following script:
+```bash
+npm install highcharts-dashboards
+```
+
+### Include Dashboards by script tag
+Include the JavaScript files in the `<head>` section of your web page as shown below.
+
+```html
+    <script src="https://code.highcharts.com/dashboard.js"></script>
+ ```
+
+
+### Load files from your domain
+In the example above the Javascript files are loaded from [code.highcharts.com](https://code.highcharts.com) but you can download the files from [highcharts.com](https://www.highcharts.com/download/) and include them on your web page. Here is the example with Highcharts served from your own server:
+
+```html
+    <script src="../code/dashboard.js"></script>
+```
+### Load additional modules
+There are some additional modules that you might want to include in your webpage, that are not included in the basic dashboard.js module. The basic module lets you create the elementary dashboard with HTMLComponents, but if you would like to use some predefined component types, you need to add some other modules e.g. to use Highcharts component, you need to add following scripts:
```

Comment:
Can we replace dashboard.js with `dashboard.js` and HTMLComponents with `HTMLComponents`. We will increase the readbility.